src/test/java/uk/gov/justice/probation/courtcasematcher/restclient/CourtCaseRestClientIntTest.java

```diff
@@ -149,15 +146,15 @@ void givenNullCaseId_whenPutCourtCase_thenItsSuccessful() {
         assertThat(voidMono.blockOptional()).isEmpty();
 
         MOCK_SERVER.verify(
-                putRequestedFor(urlMatching("/case/[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}/extended"))
+                putRequestedFor(urlMatching("/hearing/[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}"))
```

Comment:
As this regex is used in a few places you could consider breaking it out as a global constant in the test package 🤷